share/spice/octopart/buy.handlebars

```diff
@@ -1,20 +1,3 @@
-{{#if description}}
-    <div>Details: {{description}}</div>
-{{/if}}
-{{#if item.market_status}}
-    <div>Market Status: {{item.market_status}}</div>
-{{/if}}
-{{#if price}}
-    <div>Average Price: ${{price}}</div>
-{{/if}}
 <br>
-{{#if datasheet}}
-    <a href="{{{datasheet}}}" class="btn btn__datasheet">Datasheet</a>
-{{/if}}
-{{#if url}}
-    <a href="{{{url}}}" class="btn btn--primary">
-    Buy at Octopart
-    </a>
-{{/if}}
-
-
+<a href="{{{datasheet}}}" class="btn btn__datasheet">Datasheet</a>
+<a href="{{{item.detail_url}}}" class="btn btn--primary">Authorized Distributors</a>
```

Comment:
@moollaza looks like this needs to be changed too since detail_url doesn't exist. Other than that it looks good!